auth/src/main/java/org/aerogear/mobile/security/AbstractSecurityCheckExecutor.java

```diff
@@ -10,18 +10,20 @@
 import java.util.Collection;
 import java.util.HashSet;
 
+import static org.aerogear.mobile.core.utils.SanityCheck.nonNull;
+
 /**
  * Base class for security check executors.
  */
 abstract class AbstractSecurityCheckExecutor<T extends AbstractSecurityCheckExecutor> {
 
     /**
-     * Collection of checks to be executed.
+     * Collection of checks to be executed. Cannot be null.
      */
     private final Collection<SecurityCheck> checks = new HashSet<>();
 
     /**
-     * Context.
+     * Context for the device. Cannot be null.
```

Comment:
Private members do not produces javadocs.

Reply:
I'm thinking would these docs still be useful for contributors?

Reply:
Yes, but saying that the `member` can't be null is misleading. It could be that it can't be null in a constructor, but it can in another (not our case, but in general it is like this).

Reply:
Ya ok makes sense. I'll remove the `Cannot be null` the description